pkg/controller/gittrack/gittrack_controller.go

```diff
@@ -104,10 +113,12 @@ func add(mgr manager.Manager, r reconcile.Reconciler) error {
 		return err
 	}
 
-	// Watch for changes to ClusterGitTrack
-	err = c.Watch(&source.Kind{Type: &farosv1alpha1.ClusterGitTrack{}}, &handler.EnqueueRequestForObject{})
-	if err != nil {
-		return err
+	if opts.clusterGitTrackMode != farosflags.CGTMDisabled {
```

Comment:
I think this will need some more watches in it right? For the children of the ClusterGitTrack's too? We only want to watch children of ClusterGitTracks when we are managing them else we will reconcile the ClusterGitTracks any time the child objects are modified themselves

Reply:
I think this still need the other watches in? Doesn't look like anything has changed since I last reviewed?